lib/node_modules/@stdlib/utils/parse-ndjson/package.json

```
@@ -0,0 +1,67 @@
+{
+    "name": "@stdlib/utils/parse-ndjson",
+    "version": "0.0.1",
```

We handle publishing and versioning automatically and leave the version numbers at `0.0.0` in the development repo.

```suggestion
    "version": "0.0.0",
```